bee-api/bee-rest-api/src/types/responses.rs

```diff
@@ -136,9 +136,10 @@ pub struct OutputResponse {
     #[serde(rename = "isSpent")]
     pub is_spent: bool,
     #[serde(rename = "milestoneIndex")]
-    pub milestone_index: bool,
+    pub milestone_index: u32,
     #[serde(rename = "milestoneTimestamp")]
-    pub milestone_timestamp: bool,
+    // todo: u32 or u64 here? in MilestoneResponse it's an u64
+    pub milestone_timestamp: u32,
```

Comment:
Hornet uses u64 timestamps https://github.com/gohornet/hornet/blob/77ef3e0342e849e0556110f1145969b0c973ce4a/plugins/restapi/v1/types.go#L33

 (https://en.wikipedia.org/wiki/Year_2038_problem) ?

Reply:
But that's the nodeinfo, here it's an u32 https://github.com/gohornet/hornet/blob/818363b476b86203ab24db535294aea8b0afc389/plugins/restapi/v1/utxo.go#L34, they should change it then also I guess?

Reply:
So I would use u64 here